site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -167,6 +222,33 @@ gcloud container node-pools create windows \
 
 where [CLUSTER_NAME] is the name of the cluster you created.
 
+### Create an Autopilot mode cluster for Agones
+
+Agones supports Autopilot clusters in [alpha]({{<ref "/docs/Guides/feature-stages#alpha">}}).
+
+```bash
+gcloud container clusters create-auto [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION] \
+  --release-channel=[RELEASE_CHANNEL] \
+  --autoprovisioning-network-tags=game-server \
+```
+Replace the following:
+
+* **CLUSTER_NAME**: the name of your cluster.
+* **COMPUTE_REGION**: the
+  [Google Cloud region](https://cloud.google.com/compute/docs/regions-zones#available)
+  for the cluster.
+* **RELEASE_CHANNEL**: the
+  [GKE release channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions)
+  for the cluster. Can be one of `rapid`, `regular`, or `stable`. The default
+  is `regular`. The release channel must use a
+  [supported Kubernetes version]({{<ref "/docs/Installation#usage-requirements">}}).
```

Comment:
Should we note here that even though the channel has a default, other versions are available with `--cluster-version`?

The annoying thing is I can't find a page that has the equivalent data to go/gke-versions. You can infer it from channel release notes on https://cloud.google.com/kubernetes-engine/docs/release-notes, but there's not a simple table. And on Autopilot the version doesn't show up in Pantheon. So I'm not sure what to link to outside of `gcloud container get-server-config --zone us-west1`, which is.. not super friendly.

Reply:
Played with gcloud a bit and you can run  `gcloud container get-server-config --region=us-central1 --flatten=channels --filter="(channels.channel~REGULAR)" --format="yaml(channels)"` which is neater